src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
@jaredpar does that mean [this](https://github.com/dotnet/import-comments/blob/master/ImportComments/Program.cs#L45) will have to change such that it picks up the full project references?